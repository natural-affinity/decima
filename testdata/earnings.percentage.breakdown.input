decima -p 12.5 --breakdown 80
