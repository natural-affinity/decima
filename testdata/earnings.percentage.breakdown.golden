You owe: $10.00

Takehome: $70.00
Earnings: $80.00
 Percent: 12.50%
